osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs

```diff
@@ -128,13 +126,13 @@ private BeatmapVerifierContext getContext(Texture background, [CanBeNull] Stream
         /// </summary>
         /// <param name="background">The texture of the background.</param>
         /// <param name="stream">The stream representing the background file.</param>
-        private Mock<IWorkingBeatmap> getMockWorkingBeatmap(Texture background, [CanBeNull] Stream stream = null)
+        private Mock<IWorkingBeatmap> getMockWorkingBeatmap(Texture? background, Stream? stream = null)
         {
             stream ??= new MemoryStream(new byte[1024 * 1024]);
 
             var mock = new Mock<IWorkingBeatmap>();
             mock.SetupGet(w => w.Beatmap).Returns(beatmap);
-            mock.SetupGet(w => w.Background).Returns(background);
+            mock.SetupGet(w => w.Background).Returns(background.AsNonNull());
```

Comment:
Guess `AsNonNull()` can be removed again if remove the nullable annotation in the `IWorkingBeatmap` and mark Background property as nullable.